sbt

```diff
@@ -1,132 +1,124 @@
 #!/usr/bin/env bash
 #
 # A more capable sbt runner, coincidentally also called sbt.
-# Author: Paul Phillips <paulp@typesafe.com>
```

Comment:
They're required to make the build run at all. The SBT url was incorrect. I can open it as a separate PR on this branch if that's preferable.